### PR TITLE
Preliminary update to numbering and supported functions

### PR DESCRIPTION
**Pull request by janebeckman**
New files eliminate version numbers in the filename. Updated to reflect currently-supported functionality. The new graphic is a placeholder not reflecting current customization, although it shows pgAdmin 4. It  will be replaced with an updated version. These files should be regarded as preliminary updates and will be changed to reflect ongoing work.

### CODE REVIEW

**Comment by dyozie** [comment]
@janebeckman - Can you post this to a review site?  Would like to review the HTML in an actual build if possible.  Thanks!

File: gpdb-doc/dita/admin_guide/access_db/topics/g-documentation-for-pgadmin-for-greenplum-database.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE topic
+  PUBLIC "-//OASIS//DTD DITA Composite//EN" "ditabase.dtd">
+<topic id="topic8">
+        <title>Documentation for pgAdmin 4 for Greenplum Database</title>
+        <body>
+                <p>For general help on the features of the graphical interface, select <b>Help
+                                contents</b> from the <b>Help</b> menu.</p>
+                <p>For help with Greenplum-specific SQL support, select <b>Greenplum Database
```
Comment (dyozie) [inline gpdb-doc/dita/admin_guide/access_db/topics/g-documentation-for-pgadmin-for-greenplum-database.xml]: Is there actually such a menu in pgadmin iv?  I'm not sure if we customized it to that level.

File: gpdb-doc/dita/admin_guide/access_db/topics/g-documentation-for-pgadmin-for-greenplum-database.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+                                Help</b> from the <b>Help</b> menu. If you have an active internet
+                        connection, you will be directed to online Greenplum SQL reference
```
Comment (dyozie) [inline gpdb-doc/dita/admin_guide/access_db/topics/g-documentation-for-pgadmin-for-greenplum-database.xml]: Similarly - not sure if the help contains any links out into gpdb documentation - please verify this.

File: gpdb-doc/dita/admin_guide/access_db/topics/g-documentation-for-pgadmin-for-greenplum-database.xml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+                        documentation. Alternately, you can install the Greenplum Client Tools
```
Comment (dyozie) [inline gpdb-doc/dita/admin_guide/access_db/topics/g-documentation-for-pgadmin-for-greenplum-database.xml]: Almost certain the new client tools package doesn't install any documentation - need to check this.

File: gpdb-doc/dita/admin_guide/access_db/topics/g-installing-pgadmin-for-greenplum-database.xml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE topic
+  PUBLIC "-//OASIS//DTD DITA Composite//EN" "ditabase.dtd">
+<topic id="topic7">
+  <title>Installing pgAdmin 4 for Greenplum Database</title>
+  <body>
+    <p>The installation package for pgAdmin 4 for Greenplum Database is available for download from
```
Comment (dyozie) [inline gpdb-doc/dita/admin_guide/access_db/topics/g-installing-pgadmin-for-greenplum-database.xml]: Let's not call it "pgAdmin 4 for Greenplum Database" as that implies it's different from the standard pgAdmin release.  We're just using the standard pgAdmin release that's available to everyone.

File: gpdb-doc/dita/admin_guide/access_db/topics/g-pgadmin-for-greenplum-database.xml [hunk 5]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (dyozie) [inline gpdb-doc/dita/admin_guide/access_db/topics/g-pgadmin-for-greenplum-database.xml]: These topics are all pretty small - let's combine them into a single file if possible.

File: gpdb-doc/dita/admin_guide/access_db/topics/g-pgadmin-for-greenplum-database.xml [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+<!DOCTYPE topic
+  PUBLIC "-//OASIS//DTD DITA Composite//EN" "ditabase.dtd">
+<topic id="topic6">
+   <title>pgAdmin 4 for Greenplum Database</title>
+   <body>
+      <p>If you prefer a graphic interface, use pgAdmin 4. This GUI client supports PostgreSQL
+         databases with all standard pgAdmin 4 features, while adding support for some
+         Greenplum-specific features. </p>
+      <p>pgAdmin 4 supports the following Greenplum-specific features:</p>
```
Comment (dyozie) [inline gpdb-doc/dita/admin_guide/access_db/topics/g-pgadmin-for-greenplum-database.xml]: Let's get tech review for this page.  Also probably best to call out the release notes for known limitations.

File: gpdb-doc/dita/admin_guide/access_db/topics/g-pgadmin-for-greenplum-database.xml [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+      </ul>
+      <p>The following Tree Control functions are not available for Greenplum Database.</p>
+      <ul id="ul_xkb_htm_3bb">
+         <li>Casts</li>
+         <li>Event triggers</li>
+         <li>Extensions</li>
+         <li>Foreign Data Wrappers </li>
+      </ul>
+      <fig id="io149932">
+         <title>Greenplum Version of pgAdmin 4</title>
```
Comment (dyozie) [inline gpdb-doc/dita/admin_guide/access_db/topics/g-pgadmin-for-greenplum-database.xml]: Again, there's no specific "Greenplum Version" of the utility any longer.  It's just pgAdmin 4.

File: gpdb-doc/dita/admin_guide/access_db/topics/g-supported-client-applications.xml [hunk 8]
```diff
@@ -13,7 +13,7 @@
 <!--
       <li id="io155276">
         <xref href="g-pgadmin-iii-for-greenplum-database.xml#topic6"/> is an enhanced version of the
```
Comment (dyozie) [inline gpdb-doc/dita/admin_guide/access_db/topics/g-supported-client-applications.xml]: Not an enhanced version - just standard pgAdmin 4.

File: gpdb-doc/dita/admin_guide/access_db/topics/g-viewing-a-graphical-query-plan.dita [hunk 9]
```diff
@@ -3,59 +3,44 @@
 <task id="topic11">
     <title>Viewing a Graphical Query Plan</title>
     <taskbody>
-        <context>Using the pgAdmin III query tool, you can run a query with EXPLAIN to view the
+        <context>Using the pgAdmin 4 query tool, you can run a query with EXPLAIN to view the
             details of the query plan. The output includes details about operations unique to
-                Greenplum distributed query processing such as plan
-            slices and motions between segments. You can view a graphical depiction of the plan as
-            well as the text-based data output.</context>
+            Greenplum distributed query processing such as plan slices and motions between segments.
```
Comment (dyozie) [inline gpdb-doc/dita/admin_guide/access_db/topics/g-viewing-a-graphical-query-plan.dita]: Tech review to make sure this is still accurate for version 4.

File: gpdb-doc/dita/admin_guide/managing/access_db.xml [hunk 10]
```diff
@@ -91,7 +91,7 @@
           with your Greenplum installation. The <codeph>psql</codeph> client application provides an
           interactive command-line interface to Greenplum Database. </li>
         <li id="io155276"><xref href="#topic6" type="topic" format="dita"/> is an enhanced version
-          of the popular management tool pgAdmin III. Since version 1.10.0, the pgAdmin III client
+          of the popular management tool pgAdmin 4. Since version 1.10.0, the pgAdmin client
```
Comment (dyozie) [inline gpdb-doc/dita/admin_guide/managing/access_db.xml]: Not "enhanced version"

File: gpdb-doc/dita/admin_guide/managing/access_db.xml [hunk 11]
```diff
@@ -231,13 +231,12 @@
       </topic>
     </topic>
     <topic id="topic6" xml:lang="en">
-      <title id="io140208">pgAdmin III for Greenplum Database</title>
+      <title id="io140208">pgAdmin 4 for Greenplum Database</title>
```
Comment (dyozie) [inline gpdb-doc/dita/admin_guide/managing/access_db.xml]: Reword, remove "for Greenplum Database."

File: gpdb-doc/dita/admin_guide/access_db/topics/g-documentation-for-pgadmin-for-greenplum-database.xml (hunk 1 above)
Comment (janebeckman) [reply gpdb-doc/dita/admin_guide/access_db/topics/g-documentation-for-pgadmin-for-greenplum-database.xml]: I am removing the file, as it is not referenced and confuses the pull request.

File: gpdb-doc/dita/admin_guide/access_db/topics/g-viewing-a-graphical-query-plan.dita (hunk 9 above)
Comment (janebeckman) [reply gpdb-doc/dita/admin_guide/access_db/topics/g-viewing-a-graphical-query-plan.dita]: Removing this section from navigation, as graphical query is not working for GPDB at this point.

File: gpdb-doc/dita/admin_guide/access_db/topics/g-pgadmin-for-greenplum-database.xml (hunk 5 above)
Comment (janebeckman) [reply gpdb-doc/dita/admin_guide/access_db/topics/g-pgadmin-for-greenplum-database.xml]: g-performing-administrative-tasks-with-pgadmin.xml was essentially a landing page that was removed from the navigation. Deleted to clean up the directory.

**Pull request merged**